plugins/rtmp-services/data/services.json

```diff
@@ -1790,6 +1790,58 @@
                 "max video bitrate": 5000,
                 "max audio bitrate": 160
             }
+        },
+        {
+            "name": "MyFreeCams",
+            "servers": [
+                {
+                    "name": "Automatic",
+                    "url": "rtmp://publish.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "Australia",
+                    "url": "rtmp://publish-syd.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "East Asia",
+                    "url": "rtmp://publish-tyo.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "Europe (East)",
+                    "url": "rtmp://publish-buh.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "Europe (West)",
+                    "url": "rtmp://publish-ams.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "North America (East Coast)",
+                    "url": "rtmp://publish-ord.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "North America (West Coast)",
+                    "url": "rtmp://publish-tuk.myfreecams.com/NxServer"
+                },
+                {
+                    "name": "South America",
+                    "url": "rtmp://publish-sao.myfreecams.com/NxServer"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "high",
+                "supported resolutions": [
+                    "1920x1080",
+                    "1600x900",
```

Comment:
Since 900px is not evenly divisible by 8, we generally do not recommend streaming at this resolution. It's a common monitor resolution, but it does not encode as well in h.264. 1536x864 is generally more recommended.

That said, this supported resolution list is intended to be used when your backend can only support a limited number of resolutions. Is it the case that the MyFreeCams transcoding backend supports 1600x900 but does not support something like 1536x864?

Reply:
Good catch. It's not being used currently so I'll just remove it.